Havit.Blazor.Components.Web.Bootstrap/Forms/Autosuggests/AutosuggestSettings.cs

```diff
@@ -31,5 +31,10 @@ public record AutosuggestSettings : IInputSettingsWithSize
 		/// Input size.
 		/// </summary>
 		public InputSize? InputSize { get; set; }
+
+		/// <summary>
+		/// If true, the first suggestion is highlighted until another is chosen by the user.
+		/// </summary>
+		public bool? HighlightFirstSuggestion { get; set; } = true;
```

Comment:
The default value cannot be set here.
The `Settings` has to be `null` to allow the evaluation cascade.
The default (true is correct) has to be set from `HxAutosuggest` static constructor (HxAutosuggest.nongeneric.cs).